net.blumia.pineapple.steam-recording-exporter.yml

```diff
@@ -0,0 +1,89 @@
+app-id: net.blumia.pineapple.steam-recording-exporter
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: PineappleSteamRecordingExporter
+rename-icon: net.blumia.pineapple.steam-recording-exporter
+
+finish-args:
+  # X11 and Wayland support
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  
+  # Audio/Video support
+  - --socket=pulseaudio
+  - --device=dri
+  
+  # File system access
+  - --filesystem=home:ro
+  - --filesystem=~/.local/share/Steam:ro
+  - --filesystem=~/.var/app/com.valvesoftware.Steam/.local/share/Steam
+  - --filesystem=~/.steam/steam:ro
+  - --filesystem=~/Videos:rw
+  - --filesystem=~/Movies:rw
+  
+  # Network access
+  - --share=network
+  
+  # D-Bus access for desktop integration
+  - --talk-name=org.freedesktop.FileManager1
+  - --talk-name=org.gtk.vfs.*
```

Comment:
It's a qt app so why does it need a the `org.gtk.vfs` interface?

Reply:
Removed. I thought FileManager1 might related to gvfs stuff.

Done :)

Reply:
It depends on dependencies. On my device it works fine without it but I simply cannot 100% guarantee if a dependency uses it in an edge case that I didn't covered.